rpcserver.go

```diff
@@ -1943,15 +1972,14 @@ func (r *rpcServer) parseOpenChannelReq(in *lnrpc.OpenChannelRequest,
 	globalFeatureSet := r.server.featureMgr.Get(feature.SetNodeAnn)
 
 	// Ensure that a local funding amount has been specified.
```

Comment:
nit: maybe adopt the comment here, because the fundMax flag was added ?

Reply:
Or a more general question, why are we skipping all the checks if the FundMax is set. A more detailed comment would me good imo ?